server/src/tenant/metadata/metadata.module.ts

```diff
@@ -14,6 +14,7 @@ import { MigrationGeneratorModule } from './migration-generator/migration-genera
 
 const typeORMFactory = async (): Promise<TypeOrmModuleOptions> => ({
   ...typeORMMetadataModuleOptions,
+  name: 'metadata',
```

Comment:
this needs to be set in order to properly close connection:
https://github.com/typeorm/typeorm/issues/9672

Reply:
@charlesBochet Hmm strange, we've not use it since it's marked as deprecated

Reply:
If you remove it, try to run the tests (yarn test), you'll get an error. Maybe there is a cleaner way to solve this

Reply:
I'll take a look if the this deprecated prop can be replaced with something else :)